tools/deployment/com.facebook.osqueryd.plist

```diff
@@ -9,7 +9,7 @@
   <key>Label</key>
   <string>com.facebook.osqueryd</string>
   <key>Program</key>
-        <string>/usr/local/bin/osqueryd</string>
+    <string>/usr/local/bin/osqueryd --flagfile="/var/osquery/osquery.flags"</string>
```

Comment:
We should probably change `<key>Program</key>` to `<key>ProgramArguments</key>` and use:

```xml
  <key>ProgramArguments</key>
  <array>
    <string>/usr/local/bin/osqueryd</string>
    <string>--flagfile="/var/osquery/osquery.flags"</string>
  </array>
```

I don't think there's <i>really</i> a difference, but when you go past an argv[0] ProgramArguments is [recommended](https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html).